.github/workflows/ci.yml

```diff
@@ -56,9 +46,3 @@ jobs:
           
     - name: Build
       run: make ${{ matrix.targets }}
-    
-    - name: Update errata
-      run: etc/ci/update_errata.sh
- 
-    - name: Update nightlies
```

Comment:
For this PR, we should leave in the updating of nightlies, right?

Reply:
No, I think we should pause the updating until we have decided what to do and then completed the transition. The current code is already not working (well), and the redirect server is down. The updater had stopped working for more than one year and I believe we can wait a bit longer. Running the semi-working updater for nightly builds _now_ is not very useful IMO.